bot.rb

```diff
@@ -41,7 +43,27 @@ def myself
       client.user.login
     end
 
-    # Responsible for commenting to inactive issues
+    # Lock old, inactive conversations
+    def process_old(issue)
+      return if issue.locked # already locked, nothing to do here
+
+      diff_in_months = (Time.now - issue.updated_at) / 60.0 / 60.0 / 24.0 / 30.0
+
+      return if diff_in_months < ISSUE_LOCK
+
+      puts "Locking conversations for https://github.com/#{SLUG}/issues/#{issue.number} since it hasn't been updated in #{diff_in_months.round} months"
+      # Currently in beta https://developer.github.com/changes/2016-02-11-issue-locking-api/
+      cmd = "curl 'https://api.github.com/repos/#{SLUG}/issues/#{issue.number}/lock' \
+            -X PUT \
+            -H 'Authorization: token #{ENV["GITHUB_API_TOKEN"]}' \
+            -H 'Content-Length: 0' \
+            -H 'Accept: application/vnd.github.the-key-preview'"
+      `#{cmd} > /dev/null`
```

Comment:
Yeah, since it's beta there it's not possible using the API client yet. That's why I included the URL, so we can change it once it's in the "public" API